Automattic/jetpack
# Added css to hide likes and like button from print post view.

Fixes #16205

Jetpack should automatically hide extraneous elements like Likes, Sharing, and Related posts in print media. These are interactive elements and don't make sense in the context of a printed page.

In this PR, we are going to hide the "like" button and the list of users who have liked a post in the "print" view.

#### Screenshots

The post view when no one has liked a post, 1 person has liked a post and more than 1 post has liked a post is shown below. 
<img width="1039" alt="Screenshot 2021-12-07 at 7 53 59 PM" src="https://user-images.githubusercontent.com/6594561/145046597-1898c2ea-288b-45ad-9aa0-15d45fc964c9.png">

<img width="927" alt="Screenshot 2021-12-07 at 7 55 55 PM" src="https://user-images.githubusercontent.com/6594561/145046889-f85b5bed-7d63-4380-8c04-56c7fd4fdbee.png">

<img width="559" alt="Screenshot 2021-12-07 at 7 58 26 PM" src="https://user-images.githubusercontent.com/6594561/145047350-1e4f6689-adae-4a6d-9b1c-1cbee936dfdc.png">










The existing print post view when no one has liked a post, 1 person has liked a post and more than 1 post has liked a post is shown below.
<img width="534" alt="Screenshot 2021-12-07 at 7 54 08 PM" src="https://user-images.githubusercontent.com/6594561/145046647-f5d1982b-16b3-48be-a718-883fc04d7dc3.png">
<img width="557" alt="Screenshot 2021-12-07 at 7 56 50 PM" src="https://user-images.githubusercontent.com/6594561/145047099-78b336c2-ae52-4846-b2ae-638e71b602ee.png">
<img width="542" alt="Screenshot 2021-12-07 at 7 58 02 PM" src="https://user-images.githubusercontent.com/6594561/145047299-2441118e-d6ea-4ff0-8fee-7d461f882011.png">




 



This PR introduces the changes to hide the "likes" section in the print view. This is how the above screenshots would look after this PR gets merged - the likes and the list of likers will be completely hidden. 

<img width="569" alt="Screenshot 2021-12-07 at 7 55 27 PM" src="https://user-images.githubusercontent.com/6594561/145046809-321f7f76-1f9a-4eeb-8ec6-022756e0deea.png">



#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1) Create three posts such that 1 post is liked by no one, 1 post is liked by two users and 1 post is liked by a single user.
2) Go to the post view and try printing the page (you can press cmd+P to print a page on a Mac).
3) In the print view, the like section should be hidden in all three cases. 

For dotcom testing, the feature is working without the changes introduced in this PR since r236038-wpcom takes cares of it.
